fpdf/table.py

```diff
@@ -40,6 +42,7 @@ def __init__(
         padding=None,
         outer_border_width=None,
         num_heading_rows=1,
+        num_index_columns=0
```

Comment:
Given that those 2 new parameters are only required for rendering pandas dataframes, I think they should not be added there but in `PandasMixin`, even if that means overriding some `Table` methods in `PandasMixin` 🙂